sky/client/sdk.py

```diff
@@ -537,9 +557,7 @@ def launch(
         task=dag_str,
         cluster_name=cluster_name,
         retry_until_up=retry_until_up,
-        idle_minutes_to_autostop=idle_minutes_to_autostop,
```

Comment:
Should we add a TODO to deprecate these two fields in the payloads module?

Reply:
Yeah, good catch!